Wikipedia/Localizations/br.lproj/Localizable.strings

```diff
@@ -397,7 +397,7 @@
 "home-featured-article-heading" = "Pennad eus ar c'hentañ evit $1";
 "home-main-page-heading" = "Hiziv e Wikipedia";
 "home-potd-heading" = "Skeudenn an deiz evit $1";
-"home-last-update-label" = "Hizivadenn diwezhañ: %1$";
+"home-last-update-label" = "Hizivadenn diwezhañ: $1";
```

Comment:
@joewalsh 

So I think this confirms that we need to change these on TWN - i.e. changes are not getting back-propagated... Should I go change them or do we want to hack on the back-propagation bit?